src/main/CommandHandler.cpp

```diff
@@ -720,7 +720,7 @@ CommandHandler::getLedgerEntry(std::string const& params, std::string& retStr)
 
         LedgerKey k;
         fromOpaqueBase64(k, key);
-        auto le = ltx.loadWithoutRecord(k, /*loadExpiredEntry=*/false);
```

Comment:
This command will now return expired entries, when it previously didn't . I'm guessing returning expired entries is what we want, but want to make sure we didn't have a good reason for ignoring them in the first place.

Reply:
This should return expired entries. This is used for services like soroban-rpc to track state, and it would be helpful for these systems to know about expired entries. This not returning expired entries before was a bug.